packages/alice/lib/core/alice_core.dart

```diff
@@ -15,37 +15,9 @@ import 'package:alice/ui/common/alice_navigation.dart';
 import 'package:alice/utils/shake_detector.dart';
 import 'package:flutter/material.dart';
 
-typedef AliceOnCallsChanged = Future<void> Function(List<AliceHttpCall>? calls);
-
 class AliceCore {
-  /// Should user be notified with notification if there's new request caught
-  /// by Alice
-  final bool showNotification;
-
-  /// Should inspector be opened on device shake (works only with physical
-  /// with sensors)
-  final bool showInspectorOnShake;
-
-  /// Icon url for notification
-  final String notificationIcon;
-
-  /// Storage used for Alice to keep calls data.
-  final AliceStorage _aliceStorage;
-
-  /// Logger used for Alice to keep logs;
-  final AliceLogger _aliceLogger;
-
-  ///Directionality of app. If null then directionality of context will be used.
-  final TextDirection? directionality;
-
-  ///Flag used to show/hide share button
-  final bool? showShareButton;
-
-  /// Navigator key used for inspector navigator.
-  GlobalKey<NavigatorState>? navigatorKey;
-
-  /// Flag used to determine whether is inspector opened
-  bool _isInspectorOpened = false;
+  /// Configuration of Alice
+  late AliceConfiguration _configuration;
```

Comment:
Why `late`? This could just be `final`. I assume we don't expect the configuration to change, right?

Reply:
It can be changed only with this method:
```dart
  void setNavigatorKey(GlobalKey<NavigatorState> navigatorKey) {
    _configuration = _configuration.copyWith(newNavigatorKey: navigatorKey);
  }
```